### ISSUE TITLE
ghost

### ISSUE THREAD

**Issue by lipki** [SummerFields/Minetest#74]
![creatures_egg_ghost](https://cloud.githubusercontent.com/assets/401965/20450075/cb7fe6ae-aded-11e6-9601-53e8d3e9c58a.png)

**Comment by lipki** [comment]
![creatures_egg_ghost](https://cloud.githubusercontent.com/assets/401965/20450075/cb7fe6ae-aded-11e6-9601-53e8d3e9c58a.png)

**Comment by tobyplowy** [comment]
@lipki maybe the vex texture can be used for this?

**Comment by tobyplowy** [comment]
@lipki done?

**Comment by lipki** [comment]
No, I want to find a good mod of mobs, for now I find only dummies.
